src/tools/components/boundingrectitem.cpp

```diff
@@ -23,7 +23,7 @@ void BoundingRectItem::paint(QPainter *painter,
                              const QStyleOptionGraphicsItem *option,
```

Comment:
Remove this file if possible, will be overwritten by my commits anyway

Reply:
Some dependencies might break on this version, let's remove them while merging your branch instead.

Reply:
fine, merge the branch then